policy/mcs

```diff
@@ -138,6 +135,9 @@ mlsconstrain { db_database db_schema db_table db_sequence db_view db_procedure d
 mlsconstrain { db_tuple } { insert relabelto }
 	(( h1 dom h2 ) and ( l2 eq h2 ));
```

Comment:
I would think all the db classes should have the `or ( t1 != mcs_constrained_type )` condition too.

Reply:
Same as above. I pulled these directly out of Fedora's policy. The majority of these were added in: https://github.com/fedora-selinux/selinux-policy/commit/82c32d5cf4d6df42ca16513cde2b3a28df41015f